README.md

```diff
@@ -2,5 +2,13 @@
 
 ## Local setup
 
-- To run locally you will need to add two .env file which will connect to your local DBs, one for testing and one for dev-data. Add these two files in /db, naming convention is .env.[TESTDBNAME] and .env.[DEVDBNAME].
+- To run locally you will need to add two .env file which will connect to your local DBs, one for testing and one for dev-data. Add these two files in /db, naming convention is .env.test and .env.development
+- Inside the .env files you will set the DB names 
+    
+        .env.test
+        PGDATABASE=DBNAME_test 
+
+        .env.development
+        PGDATABASE=DBNAME
+
```

Comment:
Looking great. I would add that these files should be gitignored and the command to install 'dotenv'.